src/mesh/RF95Interface.cpp

```diff
@@ -67,19 +67,12 @@ bool RF95Interface::init()
 #endif
     setTransmitEnable(false);
 
-    int res = lora->begin(getFreq(), bw, sf, cr, syncWord, power, currentLimit, preambleLength);
+    int res = lora->begin(getFreq(), bw, sf, cr, syncWord, power, preambleLength);
```

Comment:
Ooooooh. That was inisidious! 
This is why I love named parameters and hate C/C++ for not having them XD